marketplace/deployer_util/deploy.sh

```diff
@@ -37,8 +37,14 @@ handle_failure() {
 }
 trap "handle_failure" EXIT
 
-NAME="$(/bin/print_config.py --param '{"x-google-marketplace": {"type": "NAME"}}')"
-NAMESPACE="$(/bin/print_config.py --param '{"x-google-marketplace": {"type": "NAMESPACE"}}')"
+NAME="$(/bin/print_config.py \
```

Comment:
Can we add a flag to `print_config` to specify mode: PRE_EXPANDED or POST_EXPANDED (default)? It simplifies the flags here, avoids repetition, and helps with documenting what's going on here.

Reply:
Can I do this as a follow up, rather than in this PR?

I'd like to unbreak master.

Reply:
SG.